external/contracts/set/BasicIssuanceModule.sol

```diff
@@ -183,11 +199,37 @@ contract BasicIssuanceModule is ModuleBase, ReentrancyGuard {
     }
 
     /**
-     * Reverts as this module should not be removable after added. Users should always
-     * have a way to redeem their Sets
+     * SET TOKEN ONLY: Allows removal (and deletion of state) of BasicIssuanceModuleV2
      */
     function removeModule() external override {
```

Comment:
This method is not protected right ? So anyone can call it ?

Reply:
Nevermind just figured out that the method would fail if the caller is not a set token 👍